buildpacks/dotnet-framework/bin/detect.bat

```diff
@@ -0,0 +1,16 @@
+@echo off
+
+set plan_path=%2
+
+:: TODO: Should this require/provide something meaningful?
```

Comment:
If there is no intention to answer this question as part of this PR then I would ask that we remove this TODO.

Reply:
Ah thanks for catching this. Just for completeness, I'll address this

Reply:
Done.